pom.xml

```diff
@@ -5,7 +5,7 @@
 	<groupId>com.avioconsulting.mule</groupId>
 	<artifactId>mule-health-check-api</artifactId>
 	<packaging>mule-extension</packaging>
-	<version>3.0.1-SNAPSHOT</version>
```

Comment:
If its already a snapshot, the version should not be changed. Keep adding to the same snapshot.

Reply:
Agreed, keep at 3.0.1 snapshot